extension/.eslintrc.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   ignorePatterns: [
     ...config.ignorePatterns,
     'src/test/fixtures/**',
-    'src/test/e2e/wdio.conf.ts',
+    'src/test/e2e/**',
```

Comment:
We have to exclude this from the ts config because it imports the ~shitty~ super awesome webdriverio expect and that completely breaks the compiler. Once we exclude it there we have to exclude it here too. It's a tradeoff I'll take. LMK if you know another way around.